vipps-ecom-api-quick-start.md

```diff
@@ -18,140 +18,80 @@ You can also perform express checkout payments and get access to user info.
 ## Table of contents
 
 * [Postman](#postman)
-  * [Prerequisites](#prerequisites)
-  * [Step 1: Get the Postman collection and environment](#step-1-get-the-postman-collection-and-environment)
-  * [Step 2: Import the Postman files](#step-2-import-the-postman-files)
-  * [Step 3: Set up Postman environment](#step-3-set-up-postman-environment)
+  * [Import Postman files and set up the environment](#import-postman-files-and-set-up-the-environment)
 * [Make API calls](#make-api-calls)
   * [A regular eCommerce payment](#a-regular-ecommerce-payment)
   * [An express checkout payment (*Vipps Hurtigkasse*)](#an-express-checkout-payment-vipps-hurtigkasse)
   * [Getting access to user info](#getting-access-to-user-info)
   * [Generating a QR code to the Vipps landing page](#generating-a-qr-code-to-the-vipps-landing-page)
-  * [Test with Force Approve](#test-with-force-approve)
 
 <!-- END_COMMENT -->
 
 ## Postman
 
-### Prerequisites
+See:
+* [Vipps quick start guides](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/quick-start-guides)
+* [The Vipps Test Environment (MT)](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/test-environment).
 
-Review
-[Vipps quick start guides](https://vippsas.github.io/vipps-developer-docs/docs/vipps-developers/quick-start-guides) for information about getting your test environment set up.
```

Comment:
This is the link to where they learn how to use Postman... 🤔

Reply:
Good point, and it's already back.